roguetown.dme

```diff
@@ -686,18 +686,17 @@
 #include "code\datums\sexcon\sex_actions\masturbate\masturbate_penis.dm"
 #include "code\datums\sexcon\sex_actions\masturbate\masturbate_penis_over.dm"
 #include "code\datums\sexcon\sex_actions\masturbate\masturbate_vagina.dm"
-#include "code\datums\sexcon\sex_actions\oral\armpit_lick.dm"
 #include "code\datums\sexcon\sex_actions\oral\blowjob.dm"
 #include "code\datums\sexcon\sex_actions\oral\crotch_nuzzle.dm"
 #include "code\datums\sexcon\sex_actions\oral\cunnilingus.dm"
```

Comment:
```suggestion
#include "code\datums\sexcon\sex_actions\oral\armpit_lick.dm"
#include "code\datums\sexcon\sex_actions\oral\blowjob.dm"
#include "code\datums\sexcon\sex_actions\oral\crotch_nuzzle.dm"
#include "code\datums\sexcon\sex_actions\oral\cunnilingus.dm"
#include "code\datums\sexcon\sex_actions\oral\foot_lick.dm"
```

Reply:
Maintainers decided pretty thoroughly that we don't want to keep adding fetish content on fetish content and bloat and already filled sexcon pannel.

Reply:
no fetish content in the literal "sexcon panel"!!!!!!!!!!!1111111

but anyway if your going to remove it at least remove it correctly this code is slop

Reply:
I meant to revert all of the removals but I evidently forgot two of them.

Reply:
The removals were specifically done to reduce bloat on the sexcon panel in consensus with the maintainer who had been reviewing the PR originally. I'm not sure whether I'll be fullmerging this/keeping it testmerged with them remaining.
I'll check with the maintainer who was the active reviewer of the original.